packages/runtime/test/transport/relationshipTemplates.test.ts

```diff
@@ -27,6 +27,38 @@ describe("RelationshipTemplate Tests", () => {
         expect(response).toBeSuccessful();
     });
 
+    test("load peer relationship template by truncated reference", async () => {
+        const uploadedTemplate = (
+            await runtimeServices1.transport.relationshipTemplates.createOwnRelationshipTemplate({
+                maxNumberOfAllocations: 1,
+                expiresAt: DateTime.utc().plus({ minutes: 10 }).toString(),
+                content: emptyRelationshipTemplateContent
+            })
+        ).value;
+
+        const result = await runtimeServices2.transport.relationshipTemplates.loadPeerRelationshipTemplate({ reference: uploadedTemplate.reference.truncated });
+        expect(result).toBeSuccessful();
+
+        const tempalte = result.value;
+        expect(tempalte.content).toStrictEqual(uploadedTemplate.content);
+    });
+
+    test("load peer relationship template by url reference", async () => {
```

Comment:
As in the test name below.

```suggestion
    test("load peer RelationshipTemplate by url reference", async () => {
```